typescript/web/src/connectors/repository/utils/list.ts

```diff
@@ -38,14 +39,18 @@ export const list =
       }
 
       const query = table.where(where);
-      const listElements = await query.sortBy(criterion);
+      const listElements = reversed
+        ? await query.reverse().sortBy(criterion)
+        : await query.sortBy(criterion);
```

Comment:
What is this for?
Can please you add a comment so that this changed block and the following ont (from l51 to 53) won't look obsure to others?

Reply:
Fixed